src/AISmart.Application/Service/TwitterService.cs

```diff
@@ -0,0 +1,66 @@
+using System;
+using System.Threading.Tasks;
+using AISmart.Agent;
+using AISmart.Agents;
+using AISmart.Agents.Group;
+using AISmart.Common;
+using AiSmart.GAgent.SocialAgent.GAgent;
+using AISmart.GEvents.Twitter;
+using AISmart.Sender;
+using AISmart.Twitter;
+using Microsoft.Extensions.Logging;
+using Orleans;
+using Volo.Abp.Application.Services;
+
+namespace AISmart.Service;
+
+public class TwitterService : ApplicationService, ITwitterService
+{
+    private readonly IClusterClient _clusterClient;
+    private readonly ILogger<TwitterService> _logger;
+
+    public TwitterService(
+        IClusterClient clusterClient,
+        ILogger<TwitterService> logger)
+    {
+        _clusterClient = clusterClient;
+        _logger = logger;
+    }
+    
+    public async Task BindTwitterAccountAsync(BindTwitterAccountDto bindTwitterAccountDto)
```

Comment:
You are creating a group of social agent and twitter agent in a generic service class? Is this only for testing?